ecr.tf

```
@@ -37,7 +37,16 @@ data "aws_iam_policy_document" "ecr_push_inline_policy" {
       "ecr:GetDownloadUrlForLayer",
       "ecr:InitiateLayerUpload",
       "ecr:PutImage",
-      "ecr:UploadLayerPart"
+      "ecr:UploadLayerPart",
+
+
+      "ecr-public:PutImage",
+      "ecr-public:InitiateLayerUpload",
+      "ecr-public:UploadLayerPart",
```

두 칸 불편합니다. 줄여주시죠(??)

```suggestion

      "ecr-public:PutImage",
      "ecr-public:InitiateLayerUpload",
      "ecr-public:UploadLayerPart",
```

---

481f589